lib/setupmanagers/qemuhck/qemu_machine.rb

```diff
@@ -358,6 +358,7 @@ def full_replacement_list
         '@client_id@' => @client_id,
         '@workspace@' => @workspace_path,
         '@memory@' => "#{option_config('memory_gb')}G",
+        '@double_memory@' => "#{2 * option_config('memory_gb')}G",
```

Comment:
Let's have `@pluggable_memory@` and `@max_memory@`; `@max_memory` will be the sum of `@memory@` and `@pluggable_memory@`.

Reply:
In this case, I need to talk with the driver developer. You can see that memory-backend-ram and virtio-mem-pci have a different memory size. What is `@pluggable_memory@` now `2G` or `1G`? That's why I don't prefer ideas with calculation.

Reply:
The same applies even now; changing the size of `mem_backend` into 512M will certainly cause a trouble. Having `@pluggable_memory@` only moves the place where the value `2G` is written, and allows calculating `@max_memory@`.

That said, I see no reason that `request-size` has to be inequal to `mem_backend` size.

Reply:
Just checked QE command line `extra_cmd= ' -object memory-backend-ram,id=vmem0,size=5G -device virtio-mem-pci,id=mem0,memdev=vmem0,node=0,requested-size=2G,indirect_desc=off,bus='+bus_num `

`request-size` != `mem_backend` size.

Reply:
It's better to ask if changing `requested-size` is OK.